app/src/main/java/com/teambeme/beme/detail/adapter/ReplyAdapter.kt

```diff
@@ -11,12 +13,13 @@ import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.teambeme.beme.R
 import com.teambeme.beme.databinding.ReplyParentBinding
-import com.teambeme.beme.detail.model.ReplyParentData
+import com.teambeme.beme.detail.model.ResponseDetail
 import com.teambeme.beme.detail.viewmodel.DetailViewModel
+import com.teambeme.beme.otherpage.view.OtherPageActivity
 
 class ReplyAdapter(private val context: Context, private val viewModel: DetailViewModel) :
     RecyclerView.Adapter<ReplyAdapter.ReplyViewHolder>() {
-    var data = mutableListOf<ReplyParentData>()
+    var comment = mutableListOf<ResponseDetail.Data.Comment>()
```

Comment:
```suggestion
    private var comment = mutableListOf<ResponseDetail.Data.Comment>()
```

객체는 웬만하면 일급 컬렉션으로 만드는게 좋아!